diff --git a/node_modules/expo-updates/scripts/create-updates-resources-ios.sh b/node_modules/expo-updates/scripts/create-updates-resources-ios.sh
index 27a58e3..85fb326 100755
--- a/node_modules/expo-updates/scripts/create-updates-resources-ios.sh
+++ b/node_modules/expo-updates/scripts/create-updates-resources-ios.sh
@@ -26,7 +26,7 @@ RCT_METRO_PORT=${RCT_METRO_PORT:=8081}
 # `$PROJECT_DIR` is passed by Xcode as the directory to the xcodeproj file.
 # in classic main project setup it is something like /path/to/app/ios
 # in new style pod project setup it is something like /path/to/app/ios/Pods
-PROJECT_DIR_BASENAME=$(basename $PROJECT_DIR)
+PROJECT_DIR_BASENAME=$(basename "$PROJECT_DIR")
 if [ "x$PROJECT_DIR_BASENAME" != "xPods" ]; then
   exit 0
 fi
